### PR TITLE
Build sparkle on Mac

### PR DESCRIPTION
**Pull request by robinbb**

### CODE REVIEW

**Comment by robinbb** [comment]
@facundominguez No need to test on Mac. Please make sure that these changes have not broken Linux.

File: shell.nix [hunk 1]
```diff
@@ -15,8 +15,7 @@ in
 haskell.lib.buildStackProject {
   name = "sparkle";
   buildInputs =
-    [ stack
-      gradle
+    [ gradle
```
Comment (facundominguez) [inline shell.nix]: git has to be included here if the inline-java repo is pointed in stack.yaml.

**Comment by facundominguez** [comment]
Besides the git glitch, the package builds fine.

**Comment by facundominguez** [comment]
The history is difficult to read. It would be best if it only includes the changes that are being contributed.

**Comment by robinbb** [comment]
I will rebase.

**Comment by robinbb** [comment]
Actually, I will clean up the branch by re-applying my changes, in another branch. Closing this PR.

**Pull request closed without merging**